anaconda-project.yml

```diff
@@ -0,0 +1,51 @@
+# After installing anaconda-project, this project can be reproduced using:
+#    anaconda-project run
+name: clifford-interact
+description: Clifford attractors with Datashader, Panel, and interact
+maintainers:
+- jbednar
+- philippjfr
+
+channels:
+- pyviz
+
+dependencies: &deps
+- python=3.6
+- notebook ==5.7.8
+- ipykernel ==5.1.0
+- nomkl
+- param ==1.9.0
+- pyviz_comms ==0.7.2
+- datashader ==0.7.0
+- bokeh ==1.1.0
+- panel ==0.5.1
+- tornado <6
+
+packages: *deps
+
+commands:
+  dashboard:
+    unix: panel serve app.ipynb
+    supports_http_options: true
+  notebook:
+    notebook: app.ipynb
+  test:
+    unix: pytest --nbsmoke-run -k *.ipynb --ignore envs
+    windows: pytest --nbsmoke-run -k *.ipynb --ignore envs
+    env_spec: test
+  lint:
+    unix: pytest --nbsmoke-lint -k *.ipynb --ignore envs
+    windows: pytest --nbsmoke-lint -k *.ipynb --ignore envs
+    env_spec: test
+
+variables: {}
+downloads: {}
+
+env_specs:
+  default: {}
+  test:
+    packages:
+    - nbsmoke ==0.2.8
+    - pytest ==4.4.1
+skip_imports:
+  environment: 5b1139e1e2f51e2fc4e7e35a4f4fb10de5a25f1f
```

Comment:
I see you encountered this as well! Is it worth deleting every time, leaving alone or maybe it actually serves a useful purpose?

Reply:
hmmm I guess this got added when I answered the prompt to indicate that I wanted to ignore the environment.yml. I don't think it serves a purpose though since I included environment.yml in my .projectignore.